src/coreclr/vm/jitinterface.cpp

```diff
@@ -11192,6 +11192,37 @@ void CEEJitInfo::BackoutJitData(EEJitManager * jitMgr)
         jitMgr->RemoveJitData(pCodeHeader, m_GCinfo_len, m_EHinfo_len);
 }
 
+/*********************************************************************/
+void CEEJitInfo::WriteCode(EEJitManager * jitMgr)
+{
+    CONTRACTL {
+        THROWS;
+        GC_TRIGGERS;
+    } CONTRACTL_END;
+
+#ifdef USE_INDIRECT_CODEHEADER
+    if (m_pRealCodeHeader != NULL)
+    {
+        // Restore the read only version of the real code header
+        m_CodeHeaderRW->SetRealCodeHeader(m_pRealCodeHeader);
+        m_pRealCodeHeader = NULL;
+    }
+#endif // USE_INDIRECT_CODEHEADER
+
+    memcpy(m_CodeHeader, m_CodeHeaderRW, m_codeWriteBufferSize);
```

Comment:
This memcpy is also unnecessary when `m_CodeHeader` and `m_CodeHeaderRW` are the same.

Reply:
Of course, I've somehow forgotten this.